.travis.yml

```diff
@@ -21,7 +21,7 @@ script:
       -w "/data" \
       ${OS_TYPE}:${OS_VERSION} \
       ./tests.sh
-  - shellcheck ood-portal-generator/sbin/update_ood_portal
+  - shellcheck -x ood-portal-generator/sbin/update_ood_portal
```

Comment:
after getting rid of the Gemfile etc. should we revert the change for adding `-x` to this shellcheck, or is this appropriate?

Reply:
It's still needed I think because other files are sourced by update_ood_portal.

Reply:
Yep, still needed:

```
$ docker run --rm -v "$PWD:/mnt" koalaman/shellcheck:stable ood-portal-generator/sbin/update_ood_portal 

In ood-portal-generator/sbin/update_ood_portal line 9:
[ -f /etc/os-release ] && source /etc/os-release
                                 ^-------------^ SC1091: Not following: /etc/os-release was not specified as input (see shellcheck -x).

For more information:
  https://www.shellcheck.net/wiki/SC1091 -- Not following: /etc/os-release wa...
```